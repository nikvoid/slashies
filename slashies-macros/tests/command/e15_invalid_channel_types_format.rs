use serenity::model::channel::PartialChannel;
use slashies_macros::Command;

/// An invalid command
#[derive(Command)]
#[name = "BadCommand"]
struct BadCommand {
    /// Field with badly formatted channel types
    #[channel_types]
    bad_field: PartialChannel,

    /// Ok field
    #[channel_types("text")]
    good_field: PartialChannel,
}

fn main() {}
