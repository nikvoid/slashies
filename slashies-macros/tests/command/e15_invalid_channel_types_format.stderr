error: Invalid "channel_types" attribute. Attribute must be of the form channel_types(type1, type2...)
 --> tests/command/e15_invalid_channel_types_format.rs:9:5
  |
9 |     #[channel_types]
  |     ^^^^^^^^^^^^^^^^
